src/main/java/Seashell.java

```diff
@@ -0,0 +1,125 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Seashell {
+
+    private ArrayList<Task> taskList;
+
+    public Seashell() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public Seashell(String saveFileText) {
+        this.taskList = new ArrayList<>();
+        // add items from saveFile
+    }
+
+    public void start() {
+        System.out.println("Hello I'm Seashell, a task manager! What can I do for you?");
+        Scanner sc = new Scanner(System.in);
+        while (true) {
+            String command = sc.nextLine();
+            try {
+                if (command.equals("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                } else if (command.equals("list")) {
+                    listTasks(taskList);
+                } else if (command.startsWith("done ")) {
+                    done(command, taskList);
+                } else if (command.startsWith("delete ")) {
+                    delete(command, taskList);
+                } else if (command.startsWith("todo")) {
+                    createTodo(command, taskList);
+                } else if (command.startsWith("deadline")) {
+                    createDeadline(command, taskList);
+                } else if (command.startsWith("event")) {
+                    createEvent(command, taskList);
+                } else {
+                    throw new SeashellException("OOPS!!! I'm sorry, but I don't know what that means :-(");
+                }
+            } catch (SeashellException e) {
+                System.out.println(e);
+            }
+        }
+    }
+
+    private void listTasks(ArrayList<Task> taskList) throws SeashellException {
+        if (taskList.isEmpty()) {
+            throw new SeashellException("OOPS!!! Task list is currently still empty!");
+        } else {
+            for (int i = 1; i <= taskList.size(); i++) {
+                Task t = taskList.get(i - 1);
+                System.out.println(i + ". " + t);
+            }
+        }
+    }
+
+    private void done(String command, ArrayList<Task> taskList) throws SeashellException {
```

Comment:
Maybe a better alternative for this method name is `MarkAsDone`? as names representing methods should be present tense verbs.